src/Directory.Build.targets

```diff
@@ -104,6 +104,16 @@
              Visible="false" Pack="false"/>
   </ItemGroup>
 
+  <Target Name="SetResourceProperties" BeforeTargets="_GetEmbeddedResourcesWithSourceGeneration">
+    <ItemGroup>
+      <EmbeddedResource Update="@(EmbeddedResource)" Condition="'%(EmbeddedResource.NuGetPackageId)' == 'Microsoft.CodeAnalysis.Collections' AND '%(FileName)' == 'Strings'">
```

Comment:
This is checking for `strings.resx` files, but I noticed we also have a `strings.shared.resx` file.

![image](https://user-images.githubusercontent.com/4691428/128908204-dcf58acf-6821-4b3a-919d-68618e4c1388.png)

Is `strings.shared.resx` relevant for this?

Reply:
I don't believe Strings.shared.resx is part of this NuGet package.